src/components/common/Directors.vue

```diff
@@ -359,39 +360,41 @@
                     />
                   </div>
 
-                  <label class="address-sub-header">Delivery Address</label>
```

Comment:
This was a bug where "Delivery Address" would display when changing a director's name.